docs/user-guide/operations/performance-tuning-tips.md

```diff
@@ -0,0 +1,161 @@
+
+
+### Using cache for object stores
+
+While using object stores, it's highly recommended to enable the object store read cache and the write cache in the storage engine. This could reduce query time by more than 10 times.
+
+The read cache stores objects or ranges on the local disk to avoid fetching the same range from the remote again. The following example shows how to enable the read cache for S3.
+- The `cache_path` is the directory to store cached objects.
+- The `cache_capacity` is the capacity of the cache.
+
+```toml
+[storage]
+type = "S3"
+bucket = "ap-southeast-1-test-bucket"
+root = "your-root"
+access_key_id = "****"
+secret_access_key = "****"
+endpoint = "https://s3.amazonaws.com/"
+region = "your-region"
+cache_path = "/path/to/s3cache"
+cache_capacity = "10G"
+```
+
+The write cache acts as a write-through cache that stores files on the local disk before uploading them to the object store. This reduces the first query latency. The following example shows how to enable the write cache.
+- The `enable_experimental_write_cache` flag enables the write cache
+- The `experimental_write_cache_size` sets the capacity of the cache
+- The `experimental_write_cache_path` sets the path to store cached files. It is under the data home by default.
+- The `experimental_write_cache_ttl` sets the TTL of the cached files.
+
+
+```toml
+[[region_engine]]
+[region_engine.mito]
+enable_experimental_write_cache = true
+experimental_write_cache_size = "10G"
+experimental_write_cache_ttl = "8h"
+# experimental_write_cache_path = "/path/to/write/cache"
+```
+
+### Enlarging cache size
+
+The `greptime_mito_cache_bytes` and `greptime_mito_cache_miss` metrics can indicate whether we need to enlarge the cache size:
+- The `greptime_mito_cache_miss` is high and increasing
+- The `greptime_mito_cache_bytes` reaches the cache capacity
+
+The `type` label in these metrics shows the type of cache. The storage engine supports configuring the size of each cache. The following is an example:
+
+```toml
+[[region_engine]]
+[region_engine.mito]
+experimental_write_cache_size = "10G"
+sst_meta_cache_size = "128MB"
+vector_cache_size = "512MB"
+page_cache_size = "512MB"
+selector_result_cache_size = "512MB"
+```
+
+### Enlarging scan parallelism
+
+The storage engine limits the number of concurrent scan tasks to 1/4 of CPU cores for each query. Enlarging the parallelism can reduce the query latency if the machine's workload is relatively low.
+
+```toml
+[[region_engine]]
+[region_engine.mito]
+scan_parallelism = 8
+```
+
+### Using append-only table if possible
+
+In general, append-only tables have a higher scan performance as the storage engine can skip merging and deduplication. We recommend enabling the [append_mode](/reference/sql/create.md#create-an-append-only-table) for the table if it doesn't require deduplication. For example, a log table should be append-only as log messages may have the same timestamp.
```

Comment:
hmm... append-only is not for performance optimization, but for the needs of specific business scenarios.
So it might not be appropriate to put it here.

Reply:
It is not only for specific business scenarios. It may improve the performance as it skips deduplication so it can scan faster. We also have some optimize rules for append-only tables. (For example, `count(*)` doesn't require reading all rows`).

I'll add these to the doc.